integration_test/tests/raw_transactions.rs

```diff
@@ -180,6 +180,49 @@ fn raw_transactions__decode_psbt__modelled() {
     // TODO: Add a taproot field and test it with v24
 }
 
+/// Test `decodepsbt` deserialization and model conversion.
+///
+/// Uses the first valid PSBT from Bitcoin Core's test/functional/data/rpc_psbt.json.
+#[test]
+fn raw_transactions__decode_psbt_core__modelled() {
+    let node = Node::with_wallet(Wallet::None, &[]);
+
+    // First valid PSBT from test/functional/data/rpc_psbt.json (valid[0]).
+    let psbt = concat!(
+        "cHNidP8BAHUCAAAAASaBcTce3/KF6Tet7qSze3gADAVmy7OtZGQXE8pCFxv2AAAAAAD+////",
+        "AtPf9QUAAAAAGXapFNDFmQPFusKGh2DpD9UhpGZap2UgiKwA4fUFAAAAABepFDVF5uM7gyxH",
+        "BQ8k0+65PJwDlIvHh7MuEwAAAQD9pQEBAAAAAAECiaPHHqtNIOA3G7ukzGmPopXJRjr6Ljl/",
+        "hTPMti+VZ+UBAAAAFxYAFL4Y0VKpsBIDna89p95PUzSe7LmF/////4b4qkOnHf8USIk6Uwpy",
+        "N+9rRgi7st0tAXHmOuxqSJC0AQAAABcWABT+Pp7xp0XpdNkCxDVZQ6vLNL1TU/////8CAMLr",
+        "CwAAAAAZdqkUhc/xCX/Z4Ai7NK9wnGIZeziXikiIrHL++E4sAAAAF6kUM5cluiHv1irHU6m8",
+        "0GfWx6ajnQWHAkcwRAIgJxK+IuAnDzlPVoMR3HyppolwuAJf3TskAinwf4pfOiQCIAGLONfc",
+        "0xTnNMkna9b7QPZzMlvEuqFEyADS8vAtsnZcASED0uFWdJQbrUqZY3LLh+GFbTZSYG2YVi/j",
+        "nF6efkE/IQUCSDBFAiEA0SuFLYXc2WHS9fSrZgZU327tzHlMDDPOXMMJ/7X85Y0CIGczio4O",
+        "FyXBl/saiK9Z9R5E5CVbIBZ8hoQDHAXR8lkqASECI7cr7vCWXRC+B3jv7NYfysb3mk6haTkz",
+        "gHNEZPhPKrMAAAAAAAAA",
+    );
+
+    let json: DecodePsbt = node.client.decode_psbt(psbt).expect("decodepsbt");
+
+    assert_eq!(json.inputs.len(), 1, "should have 1 input");
+    assert_eq!(json.outputs.len(), 2, "should have 2 outputs");
+    assert_eq!(json.psbt_version, 0, "PSBT version should be 0");
```

Comment:
```suggestion
    #[cfg(not(feature = "v22_and_below"))]
    assert_eq!(json.psbt_version, 0, "PSBT version should be 0");
```

`psbt_version` is only v23 and above